frontend_tests/node_tests/reactions.js

```diff
@@ -165,6 +165,7 @@ test("basics", () => {
             emoji_code: "1f641",
             local_id: "unicode_emoji,1f641",
             count: 1,
+            names_reacted: "Cali",
```

Comment:
I had passed another item called `names_reacted` into the context for the _message reaction view_, so I updated the tests to check for this item as well.